disruption_py/core/retrieval_manager.py

```diff
@@ -143,10 +136,7 @@ def shot_setup(
             )
             return physics_method_params
         except Exception as e:
-            self.logger.info(
-                "[Shot %s]: Caught failed to setup shot, cleaning up tree manager.",
-                shot_id,
-            )
+            logger.error(shot_log_msg(shot_id, f"Failed to setup shot! {e}"))
```

Comment:
I added the exception in-line as I believe the actual stack trace does not get added to the log file -- would it make sense to do something different here?